python/lsst/meas/algorithms/dynamicDetection.py

```diff
@@ -14,12 +14,21 @@
 from lsst.afw.table import SourceCatalog, SourceTable, IdFactory
 from lsst.meas.base import ForcedMeasurementTask
 
+import lsst.afw.image
+import lsst.afw.math
+
 
 class DynamicDetectionConfig(SourceDetectionConfig):
     """Configuration for DynamicDetectionTask"""
     prelimThresholdFactor = Field(dtype=float, default=0.5,
                                   doc="Fraction of the threshold to use for first pass (to find sky objects)")
     skyObjects = ConfigurableField(target=SkyObjectsTask, doc="Generate sky objects")
+    doBackgroundTweak = Field(dtype=bool, default=True,
+                              doc="Tweak background level so median PSF flux of sky objects is zero?")
```

Comment:
Just checking:
 - for coadds, we use `reEstimateBackground = False, doBackgroundTweak=True`
 - in processCcd, we use `reEstimateBackground = True`, and `doBackgroundTweak` isn't relevant because we're not using `DynamicDetectionTask`?

Reply:
Yes. But there's no fundamental reason why you couldn't turn both background subtractions on in `DynamicDetectionTask`.